docs/_posts/2019-09-15-seminar-meeting-minutes-3-6.md

```diff
@@ -0,0 +1,254 @@
+      width='560'
+      height='315'
+      allowfullscreen
+      webkitallowfullscreen
+      frameborder="0"
+      style="
+        position: absolute;
+        top:0;
+        left:0;
+        width:100%;
+        height:100%;
+      "
+    ></iframe>
+  </div>
+  
+- 侯正罡 @[ArronHZG](https://github.com/ArronHZG)
+
+  >语义分割中的 Attention 机制
+  >一作李夏，北大林宙辰组，本科北邮
+  >Attention 机制的分析：实际上是自相关，通过自相关对信息进行加权，关心自相关程度更大的信息。外壳结构类似于 Bottleneck。
+  >参数学习：EM 算法，提取特征的部分已经 pretrained 过，本质上是一个 Post-processing 的过程。
+  >通过实验表明了学习得到的基向量具有代表性，数量少，且互不相似，对语义分割的去噪有明显的帮助。
+  >传统方法与 DL 结合的一个案例。
+  
+  [Expectation Maximization Attention Networks for Semantic Segmentation](https://arxiv.org/abs/1907.13426)
+  [Github](https://xialipku.github.io/EMANet)
+
+  ![侯正罡]({{ '/assets/2019/seminar-3-6/seminar-3-6-arron.jpg'| relative_url }})
+
+  <div class="zoom-container" style="
+      position: relative;
+      padding-bottom:56.25%;
+      padding-top:30px;
+      height:0;
+      overflow:hidden;
+  ">
+    <iframe
+      src='{{ '/assets/js/viewer-js/' | relative_url }}#{{ '/assets/2019/seminar-3-6/1907-13426--emanet.pdf' | relative_url }}'
+      width='560'
+      height='315'
+      allowfullscreen
+      webkitallowfullscreen
+      frameborder="0"
+      style="
+        position: absolute;
+        top:0;
+        left:0;
+        width:100%;
+        height:100%;
+      "
+    ></iframe>
+  </div>
+
+### Poster
+
+- 刘云 @[fuyunfuyun666](https://github.com/fuyunfuyun666)
+  
+  >多模态问题的通用结构
+  >2 3 5 层解决不同问题，每层是一个类 Res-Block 结构，表示从低级语义到高级语义。
+  >通过实验探究了联合学习时几层 Res-Block 对问题最有帮助。
+  >但实验性过强，并没有解释结果在不同问题的语义高低方面的启示（数据处理原理，越到高层互信息越少），略显遗憾。
+
+  [Multi-task Learning of Hierarchical Vision-Language Representation](https://arxiv.org/pdf/1812.00500.pdf)
+
+  ![刘云]({{ '/assets/2019/seminar-3-6/seminar-3-6-liuyun.jpg'| relative_url }})
+
+  <div class="zoom-container" style="
+      position: relative;
+      padding-bottom:56.25%;
+      padding-top:30px;
+      height:0;
+      overflow:hidden;
+  ">
+    <iframe
+      src='{{ '/assets/js/viewer-js/#/assets/2019/seminar-3-6/1812-00500-multi-task.pdf' | relative_url }}'
+      width='560'
+      height='315'
+      allowfullscreen
+      webkitallowfullscreen
+      frameborder="0"
+      style="
+        position: absolute;
+        top:0;
+        left:0;
+        width:100%;
+        height:100%;
+      "
+    ></iframe>
+  </div>
+
+- 李卓桓 @[huan](https://github.com/huan)
+  
+  >TPU 的应用案例: [使用 TPU 训练 TensorFlow 模型（Huan）](https://www.zixia.net/tensorflow-handbook-tpu/)，帮助快速进行大规模训练
+  >可能可以申请到2500刀的资源。
+
+  ![李卓桓]({{ '/assets/2019/seminar-3-6/seminar-3-6-huan.jpg'| relative_url }})
+
+  <div class="zoom-container" style="
+      position: relative;
+      padding-bottom:56.25%;
+      padding-top:30px;
+      height:0;
+      overflow:hidden;
+  ">
+    <iframe
+      src='{{ '/assets/js/viewer-js/#/assets/2019/seminar-3-6/tensorflow-handbook-tpu.pdf' | relative_url }}'
```

Comment:
The size of TPU.pdf is more than 18MB which is too large for embedding in the webpage.

I'd like to suggest that at here we do not store the pdf file in our repository, we can link to our book url, it will be convenient enough for readers to continue reading through the links.

Reply:
OK,I already link the url in this minutes, so I can remove the PDF.